Sources/Imperial/Helpers/Sessions+Imperial.swift

```diff
@@ -38,7 +49,32 @@ extension Session {
     public func setAccessToken(_ token: String) {
         self[Keys.token] = token
     }
-    
+
+    /// Gets the refresh token from the session.
+    ///
+    /// - Returns: The refresh token stored with the `refresh_token` key.
+    /// - Throws: `Abort.unauthorized` if no refresh token exists.
+    public func refreshToken()throws -> String {
+        guard let token = self[Keys.refresh] else {
+            if self[Keys.token] == nil {
+                throw Abort(.unauthorized, reason: "User currently not authenticated")
+            } else {
+                let oauth_data = self["access_token_service"]?.data(using: .utf8) ?? Data()
```

Comment:
Why `snake_case` here for the variable names?

Reply:
Right, I was mixing up my naming conventions. Fixed.